pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -1941,3 +2005,19 @@ func (f *schemaChangeFrontier) hasLaggingSpans(
 	}
 	return frontier.Add(lagThresholdNanos, 0).Less(f.latestTs)
 }
+
+func jitter(d time.Duration) time.Duration {
+	if d == 0 {
+		return 0
+	}
+	const percent = 10
+	start, end := int64(d-d/percent), int64(d+d/percent)
+	return time.Duration(start + rand.Int63n(end-start))
+}
+
+func shouldCountBillingError(err error) bool {
```

Comment:
Q: is this an appropriate set of errors? i don't want to increment the error count when we shut down due to context canceled, which can manifest as `context.Canceled`, a grpc code = `codes.Canceled`, or a `cancelchecker.QueryCanceledError`, i think.. at least that's what i've run into while testing.

the `pgcode.UndefinedTable` one is debatable however. my reasoning is that this is going to result in a permanently failed feed anyway so 🤷 .

BUT i also don't want to mask any real errors.